Horizon-Simulations/LVFR-flybywire
# feat: Add A21N Leap ACTs

### I would like feedback if you notice something incorrect or potentially missing, also if someone could answer the questions I left in the code that would be great. This is only a draft till now.

To search for questions type the following:

- // TODO:
- // ?
- // *
- // !




## Summary of Changes


Added ACT 1, 2 and 4 to the A321 Leap

## Screenshots (if necessary)



![image](https://github.com/Horizon-Simulations/HS-flybywire/assets/81376423/0252b643-e51b-4eda-8a19-5a9b31d28cc8)


## References





## Additional context



Discord username (if different from GitHub):

## Testing instructions


I would not suggest any testing before I get the correct CSS positioning for the TankReadoutWidget for ACTs 1, 2 & 4.

Should you want to do testing:

1. Refuel the aircraft and make sure that all other tanks are filled up before ACTs start to be filled up. These should get filled in this order: ACT1, ACT2, ACT4
2. Make sure that this fuel has been loaded
3. Try to unload the fuel
4. Try a flight and see that when 11300lb (5000 kg) is reached on the centre tank, it starts to fill up from the ACTs in this order: ACT 4, ACT 2, ACT 1

Mark accordingly if they work:

- [ ] Refuel tanks
- [ ] Unload tanks
- [ ] Tanks transfer

I think that is all, if you think I missed something let me know.


## How to download the PR for QA

Every new commit to this PR will cause a new A32NX artifact to be created, built, and uploaded.

1. Make sure you are signed in to GitHub
1. Click on the **Checks** tab on the PR
1. On the left side, click on the bottom **PR** tab
1. Click on the **A32NX** download link at the bottom of the page
